<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COAC</class>
 <widget class="QMainWindow" name="COAC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>COAC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>355</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>355</width>
              <height>200</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>145</red>
                  <green>145</green>
                  <blue>145</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>145</red>
                  <green>145</green>
                  <blue>145</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>145</red>
                  <green>145</green>
                  <blue>145</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>145</red>
                  <green>145</green>
                  <blue>145</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="viewfinderPage">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QCameraViewfinder" name="viewfinder" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="previewPage">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="lastImagePreviewLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="minimumSize">
             <size>
              <width>355</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>355</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labNom">
          <property name="text">
           <string>&amp;Nom :</string>
          </property>
          <property name="buddy">
           <cstring>ledtNom</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ledtNom"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labPrenom">
          <property name="text">
           <string>&amp;Prenom :</string>
          </property>
          <property name="buddy">
           <cstring>ledtPrenom</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ledtPrenom"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labPromo">
          <property name="text">
           <string>&amp;Promo :</string>
          </property>
          <property name="buddy">
           <cstring>cmbPromos</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cmbPromos"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labAddr">
          <property name="text">
           <string>&amp;Adresse :</string>
          </property>
          <property name="buddy">
           <cstring>ledtAddr</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ledtAddr"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labVille">
          <property name="text">
           <string>&amp;Ville :</string>
          </property>
          <property name="buddy">
           <cstring>ledtVille</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="ledtVille"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labCP">
          <property name="text">
           <string>Code &amp;Postal :</string>
          </property>
          <property name="buddy">
           <cstring>ledtCP</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="ledtCP"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labMail">
          <property name="text">
           <string>&amp;Email :</string>
          </property>
          <property name="buddy">
           <cstring>ledtMail</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="ledtMail"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labSexe">
          <property name="text">
           <string>&amp;Sexe :</string>
          </property>
          <property name="buddy">
           <cstring>rdbHomme</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QRadioButton" name="rdbHomme">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>117</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>&amp;Homme</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="rdbFemme">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>0</y>
             <width>117</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>&amp;Femme</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="labDate">
          <property name="text">
           <string>&amp;Date de 
naissance :</string>
          </property>
          <property name="buddy">
           <cstring>cmbJour</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_30">
          <item>
           <widget class="QComboBox" name="cmbJour"/>
          </item>
          <item>
           <widget class="QComboBox" name="cmbMoi"/>
          </item>
          <item>
           <widget class="QComboBox" name="cmbAnnee"/>
          </item>
         </layout>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="labImg">
          <property name="text">
           <string>&amp;Images :</string>
          </property>
          <property name="buddy">
           <cstring>pbuTakeImage</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QPushButton" name="pbuTakeImage">
          <property name="text">
           <string>&amp;Choisir une Image</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QPushButton" name="pbuEnvoyer">
          <property name="text">
           <string>E&amp;nvoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>&amp;Liste</string>
    </property>
    <addaction name="actionEleve"/>
    <addaction name="actionClasse"/>
    <addaction name="actionSalle"/>
    <addaction name="actionCarte"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuAjouter">
    <property name="title">
     <string>Ajouter</string>
    </property>
    <addaction name="actionAjouterClasse"/>
    <addaction name="actionAjouterCarte"/>
    <addaction name="actionAjouterSalle"/>
    <addaction name="actionAjouterLycee"/>
   </widget>
   <addaction name="menuList"/>
   <addaction name="menuAjouter"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEleve">
   <property name="text">
    <string>&amp;Eleve</string>
   </property>
  </action>
  <action name="actionAjouterClasse">
   <property name="text">
    <string>Ajouter &amp;classe</string>
   </property>
  </action>
  <action name="actionAjouterCarte">
   <property name="text">
    <string>Ajouter c&amp;arte</string>
   </property>
  </action>
  <action name="actionAjouterSalle">
   <property name="text">
    <string>Ajouter &amp;salle</string>
   </property>
  </action>
  <action name="actionAjouterLycee">
   <property name="text">
    <string>Ajouter lycée</string>
   </property>
  </action>
  <action name="actionClasse">
   <property name="text">
    <string>&amp;Classe</string>
   </property>
  </action>
  <action name="actionSalle">
   <property name="text">
    <string>&amp;Salle</string>
   </property>
  </action>
  <action name="actionCarte">
   <property name="text">
    <string>&amp;Carte</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
